glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt

```diff
@@ -814,4 +814,15 @@ class GleanTest {
         val docType = request.path!!.split("/")[3]
         assertEquals("deletion-request", docType)
     }
+
+    @Test
+    fun `test passing in explicit BuildInfo`() {
```

Comment:
Can we also add a similar test to the [Java compatibility test](https://searchfox.org/glean/source/glean-core/android/src/test/java/mozilla/telemetry/glean/GleanFromJavaTest.java)?

Reply:
Yep.  Good idea.

Reply:
I struggled to use the Glean testing API from those Java tests, and it doesn't seem like anything else there does that already.  I filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1692506) for that, and maybe we don't need to tackle that here?

Reply:
Oh, interesting. Sure that's fine!